angular/projects/spark-angular/src/lib/components/inputs/Text.stories.ts

```diff
@@ -194,20 +194,20 @@ export const invalidHugeTextInput = () => ({
       <label sprkLabel>Huge Text Input</label>
       <span sprkFieldError>
         <sprk-icon
-          iconType="exclamation-filled-small"
+          iconType="exclamation-filled"
```

Comment:
Did we have an outdated icon name that we forgot to update with colors or was this an accident?